pkg/promtail/promtail_test.go

```diff
@@ -0,0 +1,225 @@
+package promtail
+
+import (
+	"fmt"
+	"math/rand"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/cortexproject/cortex/pkg/util"
+	"github.com/cortexproject/cortex/pkg/util/flagext"
+	"github.com/go-kit/kit/log"
```

Comment:
Can we have these in two groups (external & internal imports)? Separated with a newline.